### ISSUE TITLE
CIFAR-100 only around 20%?

### ISSUE THREAD

**Issue by lusti-Yu** [jiaxiaojunQAQ/FGSM-PGK#1]
Why is the clean training accuracy on CIFAR-100 only around 20%? @jiaxiaojunQAQ

**Comment by jiaxiaojunQAQ** [comment]
> Why is the clean training accuracy on CIFAR-100 only around 20%? [@jiaxiaojunQAQ](https://github.com/jiaxiaojunQAQ)

Thank you for your attention to our work. It may be because the parameters are not set correctly. I will find the training log and the final model checkpoints and upload them to github.

**Comment by fly2tortoise** [comment]
Dear Professor

I ran the C100 code and experienced a performance collapse. ACC is concentrated around 20% and PGD20 is nearly 12%. These numbers differ somewhat from those in the paper. Could you please share the correct parameters to run it?

Thank you very much！


  parser.add_argument('--batch-size', default=128, type=int)
  parser.add_argument('--data-dir', default='CIFAR100', type=str)
  parser.add_argument('--epochs', default=110, type=int)
  parser.add_argument('--epochs_reset', default=40, type=int)
  parser.add_argument('--lr_schedule', default='multistep', choices=['cyclic', 'multistep'])
  parser.add_argument('--lr-min', default=0., type=float)
  parser.add_argument('--lr-max', default=0.1, type=float)
  parser.add_argument('--weight-decay', default=5e-4, type=float)
  parser.add_argument('--momentum', default=0.9, type=float)
  parser.add_argument('--model', default='ResNet18', type=str, help='model name')
  parser.add_argument('--epsilon', default=8, type=int)
  parser.add_argument('--alpha', default=8, type=float, help='Step size')
  parser.add_argument('--delta-init', default='random', choices=['zero', 'random', 'previous', 'normal'], help='Perturbation')
  parser.add_argument('--normal_mean', default=0, type=float, help='normal_mean')
  parser.add_argument('--normal_std', default=1, type=float, help='normal_std')
  parser.add_argument('--out_dir', default='train_fgsm_output', type=str, help='Output directory')
  parser.add_argument('--seed', default=0, type=int, help='Random seed')
  parser.add_argument('--early-stop', action='store_true', help='Early stop if overfitting occurs')
  parser.add_argument('--factor', default=0.8, type=float, help='Label Smoothing')
  parser.add_argument('--lamda', default=1, type=float, help='Label Smoothing')
  parser.add_argument('--momentum_decay', default=0.1, type=float, help='momentum_decay')
  parser.add_argument('--EMA_value', default=0.9, type=float)
